java/solrmarc_mixin/src/de/unituebingen/ub/ubtools/solrmarcMixin/TuelibMixin.java

```diff
@@ -1615,12 +1615,57 @@ public String getSubfieldBasedSeparator(Map<String, String> separators, char sub
      * In the q-Subfield of 689 standardized subjects we have "z" (time subject), "f" (genre subject), "g" (region subject)
      * Alternatively this is in the d-Subfields
      */
-    Predicate<DataField> _689IsTimeSubject = (DataField marcField) -> {
-        if (!marcField.getTag().equals("689"))
+
+
+    Predicate<DataField> _LOK689IsTimeSubject = (DataField marcField) -> {
+        if (!(marcField.getTag().equals("LOK") && marcField.getSubfield('0') != null &&
```

Comment:
`(!(marcField.getTag().equals("LOK") && marcField.getSubfield('0') != null && marcField.getSubfield('0').getData().substring(0,3).equals("689")))`

This expression seems to be very long and is used multiple times. Can we define a shortcut for this, e.g. _IsLOKTag("689")?